azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/IWebApp.java

```diff
@@ -6,36 +6,17 @@
 package com.microsoft.azure.toolkits.appservice.service;
 
 import com.microsoft.azure.toolkits.appservice.entity.WebAppEntity;
-import com.microsoft.azure.toolkits.appservice.model.DeployType;
 import com.microsoft.azure.toolkits.appservice.model.PublishingProfile;
-import com.microsoft.azure.toolkits.appservice.model.Runtime;
 
-import java.io.File;
 import java.util.List;
 
-public interface IWebApp {
-    void start();
-
-    void stop();
-
-    void restart();
-
-    void delete();
-
-    void deploy(File file);
-
-    void deploy(DeployType deployType, File file);
-
-    boolean exists();
-
+public interface IWebApp extends IAppService {
```

Comment:
is this interface required?

Reply:
I think it may be more clear to have a interface indicated the scope, but it may be a bit over design, we could discuss it later and refactor in later PR